deprecated/README.md

```diff
@@ -1,8 +1,65 @@
 # Deprecated Deployments
 
-This folder contains deprecated versions of 1Password SCIM bridge deployments. 
+This folder contains 1Password SCIM Bridge deployment methods that have been deprecated. At the time of deprecation, these deployments are still fully functional, but will no longer be updated.
 
-At the time of deprecation, these deployments are still fully functional, but may no longer be updated and will eventually be removed.
+> 💡 **Note** that it is solely the _deployment method_ that is deprecated. Deprecating a deployment method is independent of the 1Password SCIM Bridge itself, or a specific version of the 1Password SCIM Bridge. For information about the latest version of 1Password SCIM Bridge, please see the [changelog](https://app-updates.agilebits.com/product_history/SCIM).
 
-## Deployments
- - [aws-terraform](./aws-terraform) (2020-12-21)
+## Deprecated deployment method list
+
+The following deployment methods are deprecated and will be removed from the repository on or around the Deletion Date.
+
+| Deployment                                                | Deprecation Date | Deletion Date | Suggested Alternative                                                                                                                |
```

Comment:
Do we want to link to the deprecation MR? This would serve two purposes:
 1. A place for folks to go to add comments to discuss the deprecation
 1. A place for folks to go to find the old code, so that they can fork it/use it/export it as they need.

Reply:
I think that is a great idea. Likely this would be solved along with your comment [here](https://github.com/1Password/scim-examples/pull/249#discussion_r1296024161).